### ISSUE TITLE
Selected tag can't be cleared from search field

### ISSUE THREAD

**Issue by moniuch** [Automattic/simplenote-electron#2359]
### Expected

The search input should be editable from the keyboard and react to normal caret actions. It should be rendered as regular input value. (should = this is what I expect)

### Observed

The tag text is rendered uneditable (as if HTML input placeholder). Same goes for Trash.
The only way to reset the string is to open the menu back again and click All notes.
When you start typing, your test appears, and the reset x appears. But when you empty the input, the last clicked tag is still present as the placeholder. This brings a lot of confusion as per what filter string is being applied. The expected standard behavior would be that emptying the search filter would be equivalent of "Show All Notes"

### Reproduced

1. Open menu and click on of your tags
2. Decide to view all notes unfiltered back again. Try to remove the string from search field.

### Where did you see the bug

- System Make:
- System Model:
- OS: Win
- OS version: 8.1
- Browser (if applicable): 
- Browser version (if applicable): Chrome 85
- Simplenote app version: 2.0.0-rc5 (staging.*, can't log into app.*)

**Comment by codebykat** [comment]
Yeahhhh this has always been like this and it's definitely confusing. Technically which tag is "selected" is different from the search filter. We had talked about eliminating the concept of a selected tag and making all searches global, probably something we should still do at some point.

**Comment by SylvesterWilmott** [comment]
@codebykat That's true, the search should become global and the plan is to display the currently selected tag/filter in a title bar which is something included in the UI spring cleaning project.

**Comment by moniuch** [comment]
I haven't actually used tags until only recently. My list of notes started to grow and - in the absence of folders - I needed to somehow organize them. Folders would work much better for me, as my structure is basically flat, but since they are not there yet, I tried to adopt tags and encountered a whole set of UI annoyances. I understand tags were meant as something simple to have. If there is a discussion ticket for tag brainstorming, I would love to privde my own ideas, so not to spam here.
